src/Nox.ClientApi.Tests/Tests/Controllers/CountriesController.CompoundTypes.Tests.cs

```diff
@@ -108,7 +110,41 @@ public async Task WhenPatchCompoundTypeLatLong_ShouldSucceed()
         putResult.Should().NotBeNull();
         
         putResult!.CapitalCityLocation!.Latitude.Should().Be(expectedLat);
-        putResult!.CapitalCityLocation.Longitude.Should().Be(expectedLong);
+        putResult!.CapitalCityLocation.Longitude.Should().Be(result!.CapitalCityLocation!.Longitude);
         putResult!.Name.Should().Be(expectedName);
     }
+
+    [Fact]
+    public async Task WhenPatchCompoundTypeMoney_ShouldSucceed()
+    {
+        // Arrange
+        var expectedAmount = 500000;
+        var expectedCurrencyCode = CurrencyCode.EUR;
+        var dto = new CountryCreateDto
+        {
+            Name = _fixture.Create<string>(),
+            CountryDebt = new MoneyDto { Amount = 100001, CurrencyCode = CurrencyCode.USD }
+        };
+        var result = await PostAsync<CountryCreateDto, CountryDto>(Endpoints.CountriesUrl, dto);
+
+        //Act
+        var dictionary = new Dictionary<string, object>
+        {
+            { nameof(CountryDto.CountryDebt), new Dictionary<string, object>
+                {
+                    { nameof(MoneyDto.Amount), expectedAmount },
+                    { nameof(MoneyDto.CurrencyCode), expectedCurrencyCode.ToString() }
```

Comment:
@rochar 
Delta is not deserialized when the request contains enum code instead of a string value. 
```
{
  "CountryDebt": {
    "CurrencyCode": 8
  }
}
```
This request doesn't work from Tests and from Swagger. However this one works:
```
{
  "CountryDebt": {
    "CurrencyCode": "USD"
  }
}
```
Should we take care of it now? Report it? Or maybe you know the reason?

Reply:
we already have a ticket for OdataQuery, should be the same issue. Nox-344. I will make it more priority